pkg/manifests/assets.go

```diff
@@ -0,0 +1,57 @@
+package manifests
```

Comment:
(nit) can you add the Apache license header?

Reply:
done :+1: